Pushbullet/Notifier.cs

```diff
@@ -48,26 +51,30 @@ public async Task SendNotification(UserNotification request, CancellationToken c
             var parameters = new Dictionary<string, string>
                 {
                    // {"device_iden", options.DeviceId},
+                    {"channel_tag", options.Channel},
```

Comment:
Does this work if it's null or empty?

Reply:
Yes, I have tested it.
When the string is empty, I get the notifications on all my devices, when I enter the cannel tag, it is pushed to the channel.